src/main/java/build/buildfarm/common/config/Worker.java

```diff
@@ -34,6 +34,7 @@ public class Worker {
   private boolean linkInputDirectories = true;
   private List<String> linkedInputDirectories = Arrays.asList("(?!external/)[^/]+");
   private String execOwner;
+  private List<String> execOwners = new ArrayList<>();
```

Comment:
question: could be a `Set<String>`? I assume this owner is platform-independent and i could also specify a uid (int) for unix-y OS's?

Reply:
thought: possible future enhancement: a range of uids.

Reply:
The convention here is that we have to be able to look up the user on the filesystem with https://docs.oracle.com/en/java/javase/23/docs/api/java.base/java/nio/file/attribute/UserPrincipalLookupService.html#lookupPrincipalByName(java.lang.String), and this change doesn't alter this convention from the original execOwner implementation to provide lookup by uid. I honestly don't know what would happen if we looked up that name (not in the use case). It is as platform independent as the above lookup.
This being the list of users that is presented by the queue, there's nothing conventionally keeping multiple of the same user (for god knows what reason) being presented here. A set isn't the right messaging here - you want 99% of your executions under one user, and 1% under another, this is a way to do it.
yaml's syntax permits some niceties here to deal with the ranges of names, and if we happen to work for uids, that will be true also:
`execOwners: #@ ["buildfarm-executor-{}".format(i) for i in range(1, 100)]`